.github/workflows/build.yml

```diff
@@ -467,6 +467,58 @@ jobs:
           path: swagger.json
           if-no-files-found: error
 
+  build-mssqlmigratorutility:
+    name: Build MsSqlMigratorUtility
+    runs-on: ubuntu-22.04
+    needs:
+      - lint
+    defaults:
+      run:
+        shell: bash
+        working-directory: "util/MsSqlMigratorUtility"
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - osx-x64
+          - linux-x64
+          - win-x64
+
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
+
+      - name: Print environment
+        run: |
+          whoami
+          dotnet --info
+          echo "GitHub ref: $GITHUB_REF"
+          echo "GitHub event: $GITHUB_EVENT"
+
+      - name: Restore project
+        run: |
+          echo "Restore"
+          dotnet restore
+
+      - name: Publish project
+        run: dotnet publish -c "Release" -o obj/build-output/publish -r ${{ matrix.target }} --self-contained true
+
+      - name: Upload project artifact Windows
+        if: ${{ contains(matrix.target, 'win') == true }}
+        uses: actions/upload-artifact@6673cd052c4cd6fcf4b4e6e60ea986c889389535
+        with:
+          name: MigratorCLI-${{ matrix.target }}
+          path: util/MigratorCLI/obj/build-output/publish/MigratorCLI.exe
+          if-no-files-found: error
+
+      - name: Upload project artifact
+        if: ${{ contains(matrix.target, 'win') == false }}
+        uses: actions/upload-artifact@6673cd052c4cd6fcf4b4e6e60ea986c889389535
+        with:
+          name: MigratorCLI-${{ matrix.target }}
+          path: util/MigratorCLI/obj/build-output/publish/MigratorCLI
```

Comment:
⚠️ Still some "CLI"s and I think once this runs there should be missing artifacts since these aren't the build outputs anymore given the project rename.

Reply:
Fixed :)